packages/grafana-data/src/transformations/fieldReducer.ts

```diff
@@ -320,8 +320,8 @@ export function doStandardCalcs(field: Field, ignoreNulls: boolean, nullAsZero:
 
     calcs.count++;
 
-    if (currentValue != null) {
-      // null || undefined
+    if (currentValue != null && !Number.isNaN(currentValue)) {
```

Comment:
I'm a little scared of modifying the `doStandardCalcs` function but this seems reasonable 🤷

Reply:
@ryantxu this is a change I'm a little worried about in this PR (modifying `doStandardCalcs` function) - does this seem reasonable?